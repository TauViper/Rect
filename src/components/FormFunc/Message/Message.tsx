import React, { FC } from 'react';

interface messProps {
  value: string;
  setValue: React.Dispatch<React.SetStateAction<string>>;
}
export const Message: FC<messProps> = ({ value, setValue }) => (
  <input
    className="input"
    autoFocus
    placeholder="Введите текст"
    type="text"
    value={value}
    onChange={(e) => setValue(e.target.value)}
  />
);
