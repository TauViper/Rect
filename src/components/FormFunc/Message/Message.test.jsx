import React from "react";
import { Message } from "./Message";
import { render, screen } from '@testing-library/react';

describe('Message', () => {
    it('render', () => {
        render(<Message />);
    });
    it('snapshot', () => {
        const { asFragment } = render(<Message />);
        expect(asFragment()).toMatchSnapshot();
    });
});


