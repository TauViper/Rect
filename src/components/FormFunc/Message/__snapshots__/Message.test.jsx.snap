// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot 1`] = `
<DocumentFragment>
  <input
    class="input"
    placeholder="Введите текст"
    type="text"
    value=""
  />
</DocumentFragment>
`;
